tests/unit/components/UI/Navigation/Paginator.spec.js

```diff
@@ -0,0 +1,13 @@
+import { mount } from "@vue/test-utils";
+import Paginator from "@/components/UI/Navigation/Paginator";
+
+describe("Paginator.vue", () => {
+  it("should render with required values only", () => {
+    const wrapper = mount(Paginator, {
+      props: {
+        totalItems: 10,
```

Comment:
Can store this in a variable. Will have to change at one place only if required otherwise will have to change at 2 places